### PR TITLE
Introducing changes to Support Team structure

### PR DESCRIPTION
**Pull request by olasitarska**
This PR is a result of discussions we had about the way we want to change how Django Girls Support Team works.

**Summary of changes:**
- We're introducing open, public nominations for the members of the Support Team. Everyone can nominate someone or themselves to be considered for joining the team.
- Team now performs even more of a supporting role than "decision making" role, as we feel we're growing up from the phase when any decisions need making, but instead giving more control to organizers.
- All Support Team members are now on 6 months long terms, with an option to opt-in again after that period expires. However, they're not expected to opt-in again, which makes re-evaluating commitments and self-care much easier and guilt-free. 
- Switching to having each member of the Support Team be assigned to a specific project/role ("Pull requests", "Website", "Tutorial", "T-shirts store", "Workshop Boxes").

**Things we're considering:**
- Should we have a set number of Support Team members? I feel like 6-8 people is a good number, but I cannot even come up with argument why. We can also keep it as it is now -- no precise number.

Before merging, we'd love to get community feedback on these changes, as well as hear things you think we should improve. We should also answer the two questions that are still unclear, and update the list of Support Team members who are staying for the next 6 months period.

### CODE REVIEW

File: general/sparkling.md [hunk 1]
```diff
@@ -4,38 +4,36 @@
...
 
-It's better to ask for forgiveness, than permission. If you're in the team, it means we trust you in making your own decisions and acting before asking. 
+### Well-being
 
-Do you remember when on planes they say that you should first put your oxygen mask, and then help others? We believe in the same rule. Burnout is real, and self care is more important than anything else. We're all in this for the long run, so make sure to prioritize your free time, relax and hobbies. Don't feel bad when you're sipping a drink when everyone else is working -- just enjoy it. That's why there is so many of us in the team, so that someone can enjoy the sun and the show is still running. 
+Do you remember when on planes they say that you should first put your oxygen mask, and then help others? We believe in the same rule. Burnout is real, and self care is more important than anything else. We're all in this for the long run, so make sure to prioritize your free time, relax and hobbies.
+
+To make sure it's easier for every member of the team to take care of their well-being, we've decided to introduce terms
```
Comment (aniav) [inline general/sparkling.md]: I wanted to fix that sentence but I am not sure what terms we've decided to introduce 💃

**Comment by aniav** [comment]
As a general note I really like the direction where this is going. Might be nice to hear from someone not on the Support Team.

I also feel giving each person a set of responsibilities would be easier in the terms of burnout - at some point I felt there is always something waiting for me to be done, because there is this neverending pile of e-mails and plans for the future and PRs waiting and all the other stuff 🌷

File: general/support-team.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 # About the Support Team 
-## Getting Added to the Support Team 
+## Joining the Support Team 
 1. Current support team agrees that there is a need to add a new support team member. Reasons for this include: 
   - Workload is heavy, and more team members would help reduce workload for everyone **(this is the most common reason someone is added to the support team)** 
   - New perspective on a specific issue that affects Django Girls 
   - New perspective on the general direction of Django Girls 
-2. A support team member nominates a new person. A person is usually nominated on the following basis: 
+  - Terms of past members expired, or they've decided to leave the Support Team
+2. Support Team announces a nomination period for new members, and allow all community members to nominate someone who they think would be a good addition to the team. Nominations are opened for two weeks, and are public in form of Pull Requests to Django Girls repository. Self-nominations are more than welcome! A person is usually nominated on the following basis:
```
Comment (asendecka) [inline general/support-team.md]: I'm not 100% sure if the PR are the best way to handle nominations, since I'm worried about personal data being exposed, especially when you nominate someone else.

**Comment by tBaxter** [comment]
A minor point, but rather than assigning to a specific project/role ("Pull requests", "Website", "Tutorial", "T-shirts store", "Workshop Boxes"), instead perhaps make them Directly Responsible Individual for each of those projects. In other words, it doesn't _have_ to be them doing it, but they are responsible for ensuring it is done.

**Comment by zanderle** [comment]
Sounds like a great idea. Agreed on having 6-8 people on the support team. Simply because I think that anything more than that might make the communication more difficult. 

One thing that maybe is not clear (at least to an outsider) is what falls under each project/role. I can imagine that there might be some confusion here.

And I love the idea of having opt-in to continue.  

EDIT:
+1 to what the comment before me

**Comment by phalt** [comment]
👍 for opt-out times. I think having team members around the globe or close to timezones where the majority of events could happen, will be beneficial.

Assigning specific roles is good, as it gives members focus. Those who will probably end up with a lot of work should have a secondary "helper" role too :)

I like where this is going!

**Comment by williln** [comment]
Having been on the support team before, I love these ideas (both in the PR and in the comments). Diffusion of responsibility has been a problem on the team, and letting team members self-organize into areas of responsibility with probably help with that. 

I also know community members have wanted to be able to nominate people to the support team, so having a mechanism for that is awesome. 🍒

One thing I'd love to see clarified is who makes decisions when decisions need to be made. I am +1 on giving more control to organizers, but when decisions need to be made about hiring, about using vendors, about how to spend money, etc. how are those decisions made? Majority vote among the support team, or something else?

**Comment by phalt** [comment]
@williln would a majority vote system work, given your experience on the support team? There is a line between requiring too many votes and getting stuff done. Do you think an element of "+1's" for decisions would be good? We use that in the Django CoC and so far it's been good.

**Comment by williln** [comment]
@phalt I think for most things a +1 system works fine? I do think, though, that whatever the process for making decisions is should be codified, especially if/since the support team is moving away from being decision-makers. (That makes it even more important to codify how decisions will be made when they need to be, since it will happen more rarely.) 

It might be helpful to identify a (not exclusive or exhaustive) list of things that would require a more formal vote if/when those things came up. Changes to the Code of Conduct, or to the Foundation bylaws, or additions of things that would take significantly more of the Support Team's time or the Ambassador's time. But that might also not be necessary; none of those types of things came up in the 10 months I was part of the team! I'm a natural planner!

**Comment by olasitarska** [comment]
Thanks everyone for reviewing & feedback! We appreciate your help sooo much. 

@tBaxter Good point, I'll reword this to "Directly Responsible Individual".

@zanderle Agreed! I'll add few words about each project. 

@williln @phalt Thanks for the input about decision making -- indeed an important topic. I think we will stick to +1 for now, and try to come up with a list of things that require majority vote in the next couple of weeks. It needs a little bit of thinking, so I'll branch this out as separate issue here, so we can continue moving on with this while working on decision making process in the mean time. Does that make sense? My thinking for this is that we can keep expanding this docs update forever, so I'd rather put new topics into separate discussions before this one gets too big :) @williln let me know your thoughts, and if you agree, I'll start a new issue for this.

**Comment by tBaxter** [comment]
Ola, just to ensure clarity, here's some info on the DRI principle: https://www.quora.com/How-well-does-Apples-Directly-Responsible-Individual-DRI-model-work-in-practice

**Comment by williln** [comment]
@olasitarska Sounds good!

**Comment by phalt** [comment]
@olasitarska I like it

**Comment by olasitarska** [comment]
I implemented all of the changes mentioned by you in comments now. I think we will wait for a couple of more days (let's say, until end of day anywhere on earth on Thursday) for more feedback, and then merge this and roll out nominations. 

Thanks again to everyone! ❤️

File: general/support-team-projects.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+[<- back to Who is who](people.md)
+
+## Areas that Support Team members are directly responsible for
```
Comment (patjouk) [inline general/support-team-projects.md:3]: Can we also something about Patreon? I was supposed to do it only temporally ;)

Comment (olasitarska) [inline general/support-team-projects.md:3]: @patjouk feel free to :)

**Comment by olasitarska** [comment]
Merged now. Thanks for help and feedback everyone!
